### ISSUE TITLE
core: define "the document"

### ISSUE THREAD

**Issue by pdiveris** [pdiveris/sixacts#75]
A core idea of Six Acts has been the creation of a single document, to be exposed in aURL, perhaps served also as a pdf etc. 

* If I remember well, this document will be showing the top six acts (in terms of votes.
* I guess it will have a Six Acts bannet, heading - a songle line of text then
* the proposals listed, title, category, body and below it the number of voytes..
*  I was thinking of a font like the one attached here below, for the headings, and something lighter for the body..

![Screenshot 2019-08-01 at 19 52 21](https://user-images.githubusercontent.com/1022373/62319585-33e12d00-b496-11e9-83af-51ced93468b1.png)

**Comment by EvaSchlunke** [comment]
Thank you for requesting clarification! 
Could incorporate the 'current document' view in as a filter option - not as a separate printable pdf?

What if we had 3 filter options:
Option 1. View by '_most votes_'
Option 2. View by '_most recent'_
Option 3. View '_current document_' which is a skim of the **one** top proposal from **each category**

In this context, the 'current document' view that participants can request would simply be another filtered version of the list - **one** proposal (the one with the most votes) **from each catergory** 

We can then release our own official 'snapshot' Six Acts documents at appointed times. 
This document can be one we create ourselves (not necessarily a function of the website) so that we maintain control over any printable documents which claim to be a 'Six Acts document.'

Apologies for change of instruction! This only just occured to us now, and hope it is actually a simpler way of doing it.

**Comment by pdiveris** [comment]
I am glad you asked that. I am not envisaging a separate PDF but something along the lines of what you suggested right now, a filter that produces a nice view, the contents and layout of which is what we'll be discussin here. I do, however, temd to provide a little print and or PDF icon next to either a description of a proposal (item) or within a document ("The Document.")

Have a look at this for example:

https://www.definitio.org/term/kayfabe

Check the two print and PDF buttons.

**Comment by pdiveris** [comment]
As with refards to most votes and most recent, are we still looking at fixing this to 6?

**Comment by EvaSchlunke** [comment]
Hmm, I don't think so. I can't think of any reason why not to have the **whole** list visible for _most votes_ and _most recent._

Only the 'current document' view needs to be limited to 6

The print and PDF views are nice and I think we should include these... but for the sake of maintaining control of any printable full 'Six Acts documents', maybe we can restrict these options to just individual proposals?

**Comment by pdiveris** [comment]
Yes, I agree, I too was thinking of the whole list and hoping that you'd say so too. As woth regards to print and PDF options, let's play by ear. I mean, I'll get the filter buttons and the output and then we can assess the options attached to it when in front ot us. I don't know if you did check definitio.org but if you did then you must have noticed that these are discreet buttons and they produce a reasonably good looking print.

Having said all that, at the moment I am "desigining" having no idea as to hlow long the proposals might be (but up too 100 words.)
